microshift_release_notes/microshift-4-18-release-notes.adoc

```diff
@@ -141,3 +146,23 @@ Issued: 25 February 2025
 {product-title} release 4.18.1 is now available. Bug fixes and enhancements are listed in the link:https://access.redhat.com/errata/RHEA-2024:6124[RHEA-2024:6124] advisory. Release notes for bug fixes and enhancements are provided in this documentation. The images that are included in the update are provided by the {OCP} link:https://access.redhat.com/errata/RHSA-2024:6122[RHSA-2024:6122] advisory.
 
 See the latest images included with {microshift-short} by xref:../microshift_updating/microshift-list-update-contents.adoc#microshift-get-rpm-release-info_microshift-list-update-contents[listing the contents of the {microshift-short} RPM release package].
+
+[id="microshift-4-18-1-known-issue_{context}"]
+==== Known issue
+
+* A failed greenboot health check flag does not clear when an RPM-install-based {microshift-short} service is updated without rebooting the system. As a consequence, greenboot health checks for optional components continue to fail. With this release, the primary {microshift-short} greenboot health check clears the condition that would cause optional components health checks to fail.(link:https://issues.redhat.com/browse/OCPBUGS-51198[*OCPBUGS-51198*])
```

Comment:
> With this release, the primary {microshift-short} greenboot health check clears the condition that would cause optional components health checks to fail.

I think this is true for 4.18.2, but not 4.18.1?